h2o-docs/src/product/data-science/drf.rst

```diff
@@ -36,119 +36,131 @@ H2O supports extremely randomized trees (XRT) via ``histogram_type="Random"``. W
 Defining a DRF Model
 ~~~~~~~~~~~~~~~~~~~~
 
--  `model_id <algo-params/model_id.html>`__: (Optional) Specify a custom name for the model to use as
-   a reference. By default, H2O automatically generates a destination
-   key.
+Parameters are optional unless specified as *required*.
 
--  `training_frame <algo-params/training_frame.html>`__: (Required) Specify the dataset used to build the
-   model. **NOTE**: In Flow, if you click the **Build a model** button from the
-   ``Parse`` cell, the training frame is entered automatically.
+Algorithm-specific parameters
+'''''''''''''''''''''''''''''
 
--  `validation_frame <algo-params/validation_frame.html>`__: (Optional) Specify the dataset used to evaluate
-   the accuracy of the model.
+-  `binomial_double_trees <algo-params/binomial_double_trees.html>`__: (Binary classification only) Build twice as many trees (one per class). Enabling this option can lead to higher accuracy, while disabling can result in faster model building. This option defaults to ``False`` (disabled).
 
--  `nfolds <algo-params/nfolds.html>`__: Specify the number of folds for cross-validation. This option defaults to 0 (no cross-validation).
+Shared tree-based algorithm parameters
+''''''''''''''''''''''''''''''''''''''
 
--  `y <algo-params/y.html>`__: (Required) Specify the column to use as the dependent variable. The data can be numeric or categorical.
+-  `score_tree_interval <algo-params/score_tree_interval.html>`__: Score the model after every so many trees. Disabled if set to ``0`` (default).
```

Comment:
Shouldn't we alphabetically reorder the parameters within the section? Is the current ordering completely random?

Reply:
It's not currently random; the current ordering follows along with how the R docs ([part three of the original ticket](https://h2oai.atlassian.net/browse/PUBDEV-8049)) are ordered just within each different section. I can switch it over to alphabetical instead if you're thinking that would make more sense.

Reply:
Alphabetical order is just my personal preference. I don't want enforce anything if there some ordering rule and others are ok with that. cc @wendycwong

Reply:
I personally would like alphabetical ordering because it is easier to search for the parameters.

Reply:
Noted -- reordered all open PRs to alphabetical. After I finish with the unsupervised algos, I will open a PR reordering the merged algos to alphabetical as well :)